lib/govuk_design_system_formbuilder/presenters/error_summary.rb

```diff
@@ -0,0 +1,27 @@
+module Presenters
+  # This is the default presenter for {GOVUKDesignSystemFormBuilder::Elements::ErrorSummary} and is
+  # intended to be easily replaceable should you have specific requirements that aren't met here.
+  #
+  # The basic behaviour is to always show the first error message. In Rails, error message order is
+  # determined by the order in which the validations run, but if you need to do any other transformation
+  # or concatenation, this is the place to do it.
+  class ErrorSummaryPresenter
+    # @param [Hash] error_messages the error message hash in a format that matches Rails'
+    #   `object.errors.messages`, so the format should be:
+    #
+    # @example Input format:
+    #   ErrorSummaryPresenter.new({ attribute_one: ["first error", "second error"], attribute_two: ["third error"] })
+    def initialize(error_messages)
+      @error_messages = error_messages
+    end
+
+    # Converts +@error_messages+ into an array of argument arrays that will be
+    # passed into {GOVUKDesignSystemFormBuilder::Elements::ErrorSummary#list_item}.
+    #
```

Comment:
Could add a technical definition of the return value. I notice you have this style in some technical documentation which i rather like. I am, however, not sure how an you would write the definition of an Array of Array elements that have a symbol followed by a string but here is my stab at it
```
# @return [Array<Array<Symbol, String>>] array of attribute and message arrays
```

Reply:
I always have to have a play around with this in YARD to get it working, will take a closer look